src/settings/VoucherNumber/SettingsVoucherNumberForm.js

```diff
@@ -12,6 +12,12 @@ import SettingsVoucherNumberReset from './SettingsVoucherNumberReset';
 
 import css from './VoucherNumber.css';
 
+const validatePrefix = prefix => (
+  prefix?.match(/[^a-zA-Z]/)
```

Comment:
there are could be only English letters?

Reply:
BE allows only Eng letters for voucher number

Reply:
isn't that racist?! 😆